doc/installationguide.md

```diff
@@ -10,10 +10,10 @@ These are the parameters that can be configured in the global section:
 * **contextBroker**: connection data to the Context Broker (host and port). E.g.:
```

Comment:
CNR entry missing

Reply:
fixed in d7a481626c1703026f93509bf784c4f24985f866